packages/zoe/src/zoe.chainmail

```diff
@@ -47,8 +47,9 @@ interface ZoeService {
   getInstance(instanceHandle :InstanceHandle) -> (InstanceRecord);
 
   /** 
-   * To redeem an invite, the user must provide a proposal (their rules for the
-   * offer) as well as payments to be escrowed by Zoe.
+   * To redeem an invite, the user normally provides a proposal (their rules for the
+   * offer) as well as payments to be escrowed by Zoe.  If the proposal or payments are
+   * undefined, the default is not to propose or pay (just to obtain the invite's seat).
```

Comment:
I think this text is a little confusing, but I don't have a better suggestion right now, and I know we need to do better work on explaining this process in general. So this looks good to me, I'll make an issue to better explain the process.

Reply:
If either the proposal or payments would be empty, indicate this by omitting that argument or passing `undefined`, rather than passing an empty record.

Reply:
^ The alternative sounds better to me. Thanks!

Reply:
I accepted @erights change below.